specification/addons/resource-manager/Microsoft.Addons/preview/2017-05-15/Addons.json

```diff
@@ -0,0 +1,430 @@
+                "description": "",
+                "x-ms-long-running-operation": true,
+                "x-ms-examples": {
+                    "SupportPlanTypes_Delete": {
+                        "$ref": "./examples/SupportPlanTypes_Delete.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SupportProviderName"
+                    },
+                    {
+                        "$ref": "#/parameters/CanonicalSupportPlanType"
+                    }
+                ],
+                "responses": {
+                    "202": {
+                        "description": "Accepted",
+                        "headers": {
+                        },
+                        "schema": {
+                            "$ref": "#/definitions/CanonicalSupportPlanResponseEnvelope"
+                        }
+                    },
+                    "204": {
+                        "description": "No Content"
+                    },
+                    "default": {
+                        "description": "BadRequest",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorDefinition"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Addons/supportProviders/{providerName}/supportPlanTypes":  {
+            "get": {
+                "tags": [
+                    "CanonicalSupportPlanStatus"
+                ],
+                "operationId": "CanonicalSupportPlanTypes_Get",
+                "summary": "Returns the Canonical Support Plans.",
+                "description": "Returns the Canonical Support Plans as well as whether they are enabled or not for the subscription.",
+                "x-ms-examples": {
+                    "CanonicalSupportPlanTypes_Get": {
+                        "$ref": "./examples/CanonicalSupportPlanTypes_Get.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SupportProviderName"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/CanonicalSupportPlanStatus"
```

Comment:
The response of this API should be same as the point GET API. So add the id,name and type properties here.And then add the enabled and other willemit* properties to the point GET as well. Currently the response between point GET and collection GET is different.